src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
```

Comment:
@csantanapr, git does not seem to change eols to CRLF on OS X given that `core.autocrlf=false` and/or `core.eol=crlf`. It probably could be changed by [.gitattributes only (text eol=crlf)](https://help.github.com/articles/dealing-with-line-endings/).